### ISSUE TITLE
How to say the whole day is possible?

### ISSUE THREAD

**Issue by phisad** [Copper-Head/rita_heallis#1]
I want to put the whole interval, when I dont care about the time on the day. But Then I get an error.

![image](https://user-images.githubusercontent.com/6343036/36142199-a79c9a66-10a7-11e8-9799-e95b276f88f6.png)

**Comment by phisad** [comment]
Somehow related, Rita should wait until I entered an end hour.

**Comment by phisad** [comment]
OK, looks like the documentation is misleading. Above is told to enter `9 19` but later `[9-19]` appears what makes it a bit confusing.

**Comment by iliakur** [comment]
Last night I actually also wondered how to conveniently communicate to Rita that the whole day is free. I was kind of assuming that the opposite (the whole day is booked) would be more likely so I set that as default. 
Do you think I should change the default or make the docs clearer (which I guess I'll have to do anyway).

**Comment by phisad** [comment]
Yeah, I also thought whats the better default, but thats then more personally. So maybe make it configurable in the beginning of the conversation. ^^ 

I personally would rather like to type `never` for no time at all instead of typing `9 19` for the whole day. Typing 9 19 for a whole day feels rather unnatural to me.
